pkg/gits/helpers.go

```diff
@@ -384,192 +379,185 @@ func PushRepoAndCreatePullRequest(dir string, gitInfo *GitRepository, base strin
 }
 
 // ForkAndPullPullRepo pulls the specified gitUrl into baseDir/org/repo using gitter, creating a remote fork if needed using the git provider
-func ForkAndPullPullRepo(gitURL string, baseDir string, baseRef string, branchName string, provider GitProvider, gitter Gitter, configGitFn ConfigureGitFn) (string, string, *GitRepository, error) {
+//
```

Comment:
Are the docs correct here? I mean in particular "pulls the specified gitUrl into baseDir/org/repo". The checkout/clone dir us provided via `dir`, right?